embassy-stm32/src/rcc/h.rs

```diff
@@ -585,7 +589,9 @@ pub(crate) unsafe fn init(config: Config) {
 
         RCC.ccipr5().modify(|w| {
             w.set_ckpersel(config.per_clock_source);
-            w.set_adcdacsel(config.adc_clock_source)
+            w.set_adcdacsel(config.adc_clock_source);
+            // We can't use PAC method here because H5 and H50 define different name
+            w.0 = (w.0 & !(0x03 << 8usize)) | (((config.fdcan_clock_source.to_bits() as u32) & 0x03) << 8usize);
```

Comment:
i've fixed this in https://github.com/embassy-rs/stm32-data/commit/ab2bc2a739324793656ca1640e1caee2d88df72d and #2495 , if you rebase you can use `fdcan12sel` in all h5 now.

Reply:
Fantastic. I was planning to research how to make changes in these other crates but it was a bit down on my queue.

Reply:
I pushed the change to PR to use the new method.